docker-compose-sunbird/docker-compose.yml

```diff
@@ -0,0 +1,164 @@
+      - ./data/vault-data:/vault/file
+    environment:
+      - VAULT_ADDR=${VAULT_ADDR}
+      - VAULT_API_ADDR=${VAULT_API_ADDR}
+      - VAULT_ADDRESS=${VAULT_ADDRESS}
+    cap_add:
+      - IPC_LOCK
+    command: vault server -config=/vault/config/vault.json
+    ports:
+      - 8200:8200
+    healthcheck:
+      test:
+        [
+          "CMD-SHELL",
+          "wget --spider http://127.0.0.1:8200/v1/sys/health || exit 1",
+        ]
+      interval: 10s
+      timeout: 5s
+      retries: 3
+  identity:
+    image: ghcr.io/sunbird-rc/sunbird-rc-identity-service:v2.0.0-beta2
+    ports:
+      - "3332:3332"
+    depends_on:
+      vault:
+        condition: service_healthy
+      db:
+        condition: service_healthy
+    environment:
+      - DATABASE_URL=postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db:5432/postgres
+      - VAULT_ADDR=${VAULT_ADDR}
+      - VAULT_TOKEN=${VAULT_TOKEN}
+      - VAULT_BASE_URL=${VAULT_BASE_URL}
+      - VAULT_ROOT_PATH=${VAULT_ROOT_PATH}
+      - VAULT_TIMEOUT=${VAULT_TIMEOUT}
+      - VAULT_PROXY=${VAULT_PROXY}
+      - SIGNING_ALGORITHM=${SIGNING_ALGORITHM}
+      - JWKS_URI=${JWKS_URI}
+      - ENABLE_AUTH=${ENABLE_AUTH}
+    healthcheck:
+      test:
+        [ "CMD-SHELL", "curl -f http://localhost:3332/health || exit 1" ]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+  schema:
+    image: ghcr.io/sunbird-rc/sunbird-rc-credential-schema:v2.0.0-beta1
+    ports:
+      - "3333:3333"
+    depends_on:
+      db:
+        condition: service_healthy
+      identity:
+        condition: service_healthy
+    environment:
+      - DATABASE_URL=postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db:5432/postgres
+      - IDENTITY_BASE_URL=${IDENTITY_BASE_URL}
+      - JWKS_URI=${JWKS_URI}
+      - ENABLE_AUTH=${ENABLE_AUTH}
+    healthcheck:
+      test:
+        [ "CMD-SHELL", "curl -f http://localhost:3333/health || exit 1" ]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+  credential:
+    image: ghcr.io/sunbird-rc/sunbird-rc-credentials-service:v2.0.0-beta2
+    ports:
+      - "3000:3000"
+    depends_on:
+      db:
+        condition: service_healthy
+      identity:
+        condition: service_healthy
+      schema:
+        condition: service_healthy
+    environment:
+      - DATABASE_URL=postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db:5432/postgres
+      - IDENTITY_BASE_URL=${IDENTITY_BASE_URL}
+      - SCHEMA_BASE_URL=${SCHEMA_BASE_URL}
+      - CREDENTIAL_SERVICE_BASE_URL=${CREDENTIAL_SERVICE_BASE_URL}
+      - JWKS_URI=${JWKS_URI}
+      - ENABLE_AUTH=${ENABLE_AUTH}
+    healthcheck:
+      test:
+        [ "CMD-SHELL", "curl -f http://localhost:3000/health || exit 1" ]
+      interval: 10s
+      timeout: 5s
+      retries: 5
+  db:
+    image: postgres:12.1-alpine
+    ports:
+      - 5433:5432
+    environment:
+      - POSTGRES_USER=${POSTGRES_USER}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+    volumes:
+      - ./db-data:/var/lib/postgresql/data
```

Comment:
can we keep all the mounts under a single folder, so if someone wants a new installation we can simply create that folder and we are good ?

Reply:
used a single folder for volume mounts